front/src/components/celebrities/celebrities.component.js

```diff
@@ -0,0 +1,92 @@
+export default {
+  name: 'ff-celebrities',
+  data: function () {
+    return {
+      celebrities: [
+        {
+          FirstName: 'Vlad',
+          LastName: 'Volkovskyi',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1m',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Роберт',
+          LastName: 'Дауни (младший)',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1000k',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Роберт',
+          LastName: 'Дауни (младший)',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1000k',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Роберт',
+          LastName: 'Дауни (младший)',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1000k',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.02m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.5m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.5m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.5m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
```

Comment:
it is better to put in images directory

Reply:
in assets

Reply:
This image is just for testing while the backend isn`t applied

Reply:
anyway))